resources/validations/ui/src/context/browser/views/components/SSPValidator.tsx

```diff
@@ -16,16 +17,34 @@ export const SSPValidator = () => {
         <span className="usa-hint" id="file-input-specific-hint">
           Select XML file
```

Comment:
I would recommend we put something in the UI that is a little more contextually relevant for people not read into the purpose of the application.

I know this part of the UI and UX here, and it seems the `Drag file here` is from the USWDS, so I am not sure we want to customize that. I assume the headache is not pleasant but can we move the content underneath for the default file.

I am not sure this looks more busy, but I would love more feedback from the team. I just played with Firefox to edit the DOM in place after much trial and error.

Before:
<img width="998" alt="Screen Shot 2021-06-24 at 11 47 30 AM" src="https://user-images.githubusercontent.com/61464190/123293420-f6e70f00-d4e1-11eb-8e7d-7d7c340b5ebc.png">


After:
<img width="828" alt="Screen Shot 2021-06-24 at 11 46 22 AM" src="https://user-images.githubusercontent.com/61464190/123293225-d028d880-d4e1-11eb-83e4-23b34314ca26.png">

Thoughts @GaryGapinski and @rrkane10x?

Reply:
I tried this with the supplied document, one that I supplied with a specific schema association and one without an explicit schema association. All produced good output (and the schema association was properly ignored).

I noticed the results screen shows a few superfluous roles (warn, warning, info, information, unspecified). I have made these consistent (information, warning, error, and fatal) in #108.